filters/ema.hpp

```diff
@@ -63,6 +56,9 @@ class ExponentialMovingAverage {
 
  private:
   //! Constant describing how the window weights decrease over time.
+  //! It controls how long is the moving average period.
```

Comment:
s/how long is the moving average period./how long the moving average period is./